javacord-api/src/main/java/org/javacord/api/entity/user/User.java

```diff
@@ -354,7 +354,7 @@ default String getDiscriminatedName() {
     /**
      * Mutes this user on the given server.
      *
-     * @param server The server to umute this user on.
+     * @param server The server to unmute this user on.
```

Comment:
This contradicts the method.
```suggestion
     * @param server The server to mute this user on.
```